core/Piranha/Services/IMediaService.cs

```diff
@@ -93,6 +102,8 @@ public interface IMediaService
         /// <returns>The public URL</returns>
         Task<string> EnsureVersionAsync(Guid id, int width, int? height = null);
 
+        Task<string> EnsureVersionAsync(Media media, int width, int? height);
```

Comment:
Shouldn't height be optional here like in the method above?

Reply:
Whoops! That was my mistake.